.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 20.x
-      - run: yarn install --frozen-lockfile
+      - run: yarn install --frozen-lockfile --ignore-engines
```

Comment:
Do we really need this?

Reply:
Yes, because the CI does not pass on Node 16 anymore without it, due to `@antfu/eslint-define-config` (required by the new ESLint config) requiring a newer engine version. It can be removed again after Node 16 is dropped from the CI.

Reply:
Ok, let's drop Node 16 from the CI.